src/components/Accordion/Accordion.spec.tsx

```diff
@@ -0,0 +1,93 @@
+describe('Card', () => {
+    const types = [
+        'base-accordion',
+        'single-varian-accordion',
+    ]
+
+    types.map((type) => {
+        it(`${type}: Should render the accordion`, () => {
+            cy.visit(
+                `${Cypress.env(
+                    'baseUrl'
+                )}/iframe.html?id=components-accordion--${type}`
+            )
+            cy.get(`.c-accordion`).should('exist')
+        })
+    })
+
+    types.map((type) => {
+        it(`${type}: Should render the input`, () => {
+            cy.visit(
+                `${Cypress.env(
+                    'baseUrl'
+                )}/iframe.html?id=components-accordion--${type}`
+            )
+            cy.get(`input`).should('exist')
+        })
+    })
+
+    types.map((type) => {
+        it(`${type}: Should render the accordion input`, () => {
+            cy.visit(
+                `${Cypress.env(
+                    'baseUrl'
+                )}/iframe.html?id=components-accordion--${type}`
+            )
+            cy.get(`.accordion__input`).should('exist')
+        })
+    })
+
+    types.map((type) => {
+        it(`${type}: Should render the label`, () => {
+            cy.visit(
+                `${Cypress.env(
+                    'baseUrl'
+                )}/iframe.html?id=components-accordion--${type}`
+            )
+            cy.get(`label`).should('exist')
+        })
+    })
+
+    types.map((type) => {
+        it(`${type}: Should render the accordion content`, () => {
+            cy.visit(
+                `${Cypress.env(
+                    'baseUrl'
+                )}/iframe.html?id=components-accordion--${type}`
+            )
+            cy.get(`.accordion__content`).should('exist')
+        })
+    })
+
+    types.map((type) => {
+        it(`${type}: Should render the accordion spacing`, () => {
+            cy.visit(
+                `${Cypress.env(
+                    'baseUrl'
+                )}/iframe.html?id=components-accordion--${type}`
+            )
+            cy.get(`.accordion__spacing`).should('exist')
+        })
+    })
+
+    types.map((type) => {
+        it(`${type}: Should render the paragraph`, () => {
+            cy.visit(
+                `${Cypress.env(
+                    'baseUrl'
+                )}/iframe.html?id=components-accordion--${type}`
+            )
+            cy.get(`p`).should('exist')
+        })
+    })
+
+    //Single variant accordion
+    it(`Single variant accordion: Should render accordion input single`, () => {
+        cy.visit(
+            `${Cypress.env(
+                'baseUrl'
+            )}/iframe.html?id=components-accordion--single-varian-accordion`
+        )
+        cy.get(`.accordion__input--single`).should('exist')
+    })
+})
```

Comment:
Please add tests to check if the accordion is being open when click on it. 
Check the aria states as well.

Reply:
Done, test for both checkbox is checked and unchecked for input, label and div for aira-expanded and aira-hidden

Reply:
Good job. 
One small change. Please do the check before the uncheck :)

Reply:
Done, check first, then unchecked test